### ISSUE TITLE
Resources for Open Science in Astronomy (ROSA)

### ISSUE THREAD

**Issue by aariops** [mozilla/global-sprint#242]
**[ Project Contact ]** @rainsworth
**[ GitHub Repo ]** https://github.com/rainsworth/ROSA/
**[ Track ]** Openness, 
**[ Location ]** Manchester, UK - BST (GMT+1)
**[ Coach ]** @SamanthaHindle & @aariops

### Description
Our mission is to compile and tailor open science best practices from around the web into a resource kit for astronomers to work openly from proposal to publication - we want to supply a single resource to early career scientists to empower them with the tools to practice open access/data/source/science! 
***

## Want to contribute to this project during #mozsprint?
Join us at the Global Sprint, May 10-11. Leave a comment below if you're interested in contributing to this project during #mozsprint 2018!

***

## Note to the Project Lead :tada:
Congrats, @rainsworth! This is your official project listing for the Mozilla Global Sprint 2018. To confirm your registration, please complete and check off the following:

- [x] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Provide a GitHub repository for work and discussion on your project (in a comment or at the top of this issue)
- [x] [Create a README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [x] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [x] Turn on your [Issue Tracker](https://mozilla.github.io/global-sprint/project-lead-guide/templates/#4-create-issues) and create at least three issues to describe each task that you need help with and how a contributor can get started on that task.
- [x] [Create a label](https://mozilla.github.io/global-sprint/project-lead-guide/templates/#5-add-a-mozsprint-label) called `mozsprint` and apply it to your issues.

#### Checklist for FEATURED Projects :clipboard:
To have your project FEATURED on [Mozilla Pulse](http://mozillapulse.org/), complete the following documentation. In past Sprints, well-documented featured projects have 5 times more contributions than other projects. Details about each item and more information about how to create them are [on our Project Requirements Page](https://mozilla.github.io/global-sprint/project-lead-guide/project-requirements/).

* [x] In your README, link to the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) or [write your own code of conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/).
* [x] Create file: CONTRIBUTING.md so others know how they can contribute. If you'd like, you can [remix this template](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md)
* [ ] [Follow these instructions](https://mozilla.github.io/global-sprint/project-lead-guide/featured-projects/) to submit your project to Mozilla Pulse.

Once all of the above is complete,
- [ ] Leave a comment with the text `This is ready to be featured on Mozilla Pulse`. Your Project Coach  will review this issue and approve your project if everything looks good :balloon:

If you get stuck at any point, feel free to look at the [requirements page](https://mozilla.github.io/global-sprint/project-lead-guide/project-requirements/) and [project templates](https://mozilla.github.io/global-sprint/project-lead-guide/templates/) or reach out to your Project Coach, @SamanthaHindle & @aariops. We're here to help you through this process.

**Comment by aariops** [comment]
Super @rainsworth, 

Your project is almost ready to be featured! :smile: All you need is to follow the instructions for Pulse. If you need help, let me or @SamanthaHindle know. 

Just a small thing: I'd remove the `hacktoberfest` tag from the issues. Also, beyond discussions, are there any specific issues you'd like people to help you with during the sprint - content, translations, design, code? That way, it'll be easier for participants to get involved. 

Let's get the fun started! :rocket:

**Comment by rainsworth** [comment]
Hi @aariops !

ROSA is already featured on Mozilla Pulse from #mozfest (https://www.mozillapulse.org/entry/460). Should I still follow the instructions to make another entry?

I've also removed the ```hacktoberfest``` tag. I will also tidy up the tags a bit over the coming weeks and make the repo more #mozsprint friendly!

Thanks!

**Comment by abbycabs** [comment]
Yes please make a new entry on pulse with the new tags for this year!

**Comment by rainsworth** [comment]
@aariops @SamanthaHindle This is ready to be featured on Mozilla Pulse!

**Comment by abbycabs** [comment]
Just noticed that this is a duplicate of #234! Closing this one -- @aariops @SamanthaHindle please review in #234
